docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+**Example:**
+
+`     PREGENDER:M    `
+
+Character's gender must start with "M".
+
+------------------------------------------------------------------------
+
+### PREHANDS
+
+Requires a creature to have a certain number of hands.
+
+See also <a href="#prelegs">PRELEGS</a>.
+
+#### Syntax
+
+`     PREHANDSoperator:number_of_hands    `
+
+`     operator    ` is a comparison operator from the list `     EQ    ` , `     LT    ` , `     LTEQ    ` , `     GT    ` , `     GTEQ    ` , or `     NEQ    ` . The comparison operator is written as part of the tag name, i.e. `     PREHANDSLTEQ    ` . See [comparison operators](#comparison-operators) for more details.
+
+The `     operator    ` is written as part of the tag name, i.e. `     PREHANDSNEQ    ` or `     PREHANDSGTEQ    ` .
+
+`number_of_hands` is the required number of hands.
+
+#### Examples
+
+`     PRELEGSGTEQ:4    `
+
+Character must have at least 4 hands.
+
+------------------------------------------------------------------------
+
+<span id="prehd"></span> \*\*\* Updated 5.13.5
+
+**Tag Name:** PREHD:x,y
+
+**Variables Used (x):** MIN=Number or Formula (Minimum racial Hit Dice)
+
+**Variables Used (y):** MAX=Number or Formula (Maximum racial Hit Dice)
+
+**What it does:**
+
+-   Provides the number, minimum or maximum, of hit dice the character must have to qualify.
+-   This test looks at a character's racial HD and/or any levels of Monster Classes only.
+
+**Examples:**
+
+`     PREHD:MIN=1,MAX=3    `
+
+The character must have one, two, or three racial hit die monster levels.
+
+`     PREHD:MIN=4    `
+
+The character must have four or more racial hit die or monster levels.
+
+`     PREHD:MAX=4    `
+
+The character must have four or fewer racial hit die monster levels.
+
+`     !PREHD:MIN=5,MAX=7    `
+
+Character must have between 0-4 OR 7 or greater racial hit die or monster levels.
+
+------------------------------------------------------------------------
+
+<span id="prehp"></span> \*\*\* New 5.10
+
+**Tag Name:** PREHP:x
+
+**Variables Used (x):** Number (minimum Hitpoints)
+
+**What it does:**
+
+Makes the amount of hitpoints a character has a prerequisite.
+
+**Examples:**
+
+`     PREHP:50    `
+
+The character must have at least 50 hp.
+
+------------------------------------------------------------------------
+
+**<span id="preitem"></span> Tag Name:** PREITEM:x,y,y
+
+**Variables Used (x):** Number (The number of items a character must possess).
+
+**Variables Used (y):** Text (The name of an item a character must possess - "%" may be used as a wildcard).
+
+**Variables Used (y):** TYPE=Text (The type of an item the character must possess).
+
+**What it does:**
+
+Sets requirements for items a character must possess.
+
+TYPE.xxx for reference, TYPE=xxx for assignment.
+
+**Examples:**
+
+`     PREITEM:1,Sword (Long),Sword (Short)    `
```

Comment:
Longsword exists, not sure about Sword (Long). Just as a note, Object (Deal) is a substitution for Object, Deal. Since we cannot use commas in names due to commas being separators. Thus Sword, Short exists.